src/content/en/2021/privacy.md

```diff
@@ -5,572 +5,541 @@ description: Privacy chapter of the 2021 Web Almanac covering adoption and impac
 
 ### CNAME tracking
 
-Continuing with techniques that circumvent blocks on third-party tracking, <a hreflang="en" href="https://medium.com/nextdns/cname-cloaking-the-dangerous-disguise-of-third-party-trackers-195205dc522a">CNAME tracking</a> is a novel approach where a first-party subdomain redirects to a third-party tracker using a CNAME record at the [DNS level](https://adguard.com/en/blog/cname-tracking.html). From the viewpoint of the browser, everything happens within a first-party context, so none of the third-party countermeasures are applied. Major tracking companies such as Adobe and Oracle are <a hreflang="en" href="https://sciendo.com/article/10.2478/popets-2021-0053">already offering CNAME tracking solutions</a> to their customers.
-
-The most popular company performing CNAME-based tracking is <a hreflang="en" href="https://www.pardot.com/">Pardot</a>, which is present on 6619 desktop websites. Also notable in size is Adobe, with 2671 publisher domains.
-
-
+Continuing with techniques that circumvent blocks on third-party tracking, <a hreflang="en" href="https://medium.com/nextdns/cname-cloaking-the-dangerous-disguise-of-third-party-trackers-195205dc522a">CNAME tracking</a> is a novel approach where a first-party subdomain masks the use of a third-party service using a CNAME record at the [DNS level](https://adguard.com/en/blog/cname-tracking.html). From the viewpoint of the browser, everything happens within a first-party context, so none of the third-party countermeasures are applied. Major tracking companies such as Adobe and Oracle are <a hreflang="en" href="https://sciendo.com/article/10.2478/popets-2021-0053">already offering CNAME tracking solutions</a> to their customers.
 
 {{ figure_markup(
   image="nb_sites_with_cname_tracking.png",
   caption="Websites using CNAME-based tracking on a desktop client",
-  description="Chart showing the number of websites that use a CNAME-based tracker, ordered by the popularity of the tracker.",
+  description="Chart showing the number of websites that use a CNAME-based tracker, ordered by the popularity of the tracker. Pardot is used on 6,619 sites, Adobe Experience Cloud on 2,671, Act-On Software on 869, Oracle Eloqua on 442, Eulerian on 213, Webtrekk on 101, Ingenious Technologies on 40, AT Internet on 34, Tracedock on 28, Wizaly on 15, Criteo on 15, and finally Keyade on 11.",
   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vRfOwcNkLp-mYBkmhDV5AFxl8p0ls9oxFnmmo0WUcAJxjxQqmHjquRZCWj_lNZRyFtX3RdH5T92IESu/pubchart?oid=1887798671&format=interactive",
   sheets_gid="1286114815",
   sql_file="nb_sites_with_cname_tracking.sql"
   )
 }}
 
+{# TODO - this SQL doesn't exist. Also why only desktop? #}
```

Comment:
We are missing the CNAME SQL queries, meaning I couldn't convert to percentage

Reply:
@ydimova